### ISSUE TITLE
Detected call of `lr_scheduler.step()` before `optimizer.step()`.

### ISSUE THREAD

**Issue by michael-conrad** [DigitalPhonetics/IMS-Toucan#18]
```python
lr_scheduler.py:129:
    UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`.
    In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.
    Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.
    See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
```

**Comment by Flux9665** [comment]
This is a meaningless warning that torch triggers incorrectly. The calls happen in the correct order, but because we use a custom scheduler, torch doesn't get it. Just ignore.
